src/components/Registration/BusinessNumber.vue

```diff
@@ -13,7 +13,6 @@
           x-small
           label
           color="primary"
-          text-color="white"
```

Comment:
Probably need to revert this.

V3 pages show it being used:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/e33bfd98-e4d8-40a5-8c41-db401f6c5774)

Reply:
Hmmm, that's weird. Lint was complaining about it as a deprecated prop. 
![BusinessNumber](https://github.com/bcgov/business-create-ui/assets/122301442/cc352a8e-41cd-4379-b41b-88b6b8b116a2)

I'll have a look at a alternative.

Reply:
So their own example uses a deprecated prop? LOL

By the way, "deprecated" doesn't mean you can't use it; it means they're going to obsolete it at some point.

Reply:
You don't need to solve this today. Just add a note that this needs to be fixed in your comments.

Anyway, I expect that we'll look at these code changes again when we pick up where you left off.

Reply:
Sev, I've written down all the stuff that need verification later on:
![things that need to be verified](https://github.com/bcgov/business-create-ui/assets/122301442/b79442c0-ec3c-48cf-b5df-2fe4f5ce4813)

I'll try to fix them with the links that you provided (I really appreciate it). We'll eventually be verifying them and seeing the changes later on when we get it up and running properly.

Reply:
You're one step ahead of me!

Sounds good. Good job!

Reply:
This again, is super weird to me. I'm literally on their site and chose the exact same versions in here:
![vuetify lint error](https://github.com/bcgov/business-create-ui/assets/122301442/960fd02b-44fa-4c13-bc59-beeb90c74f7a)

I can text-color being used. I reverted the change and lint is complaining about it haha.